### PR TITLE
fix a few things in the readme

### PR DESCRIPTION
**Pull request by bpownow**

### CODE REVIEW

**Comment by bpownow** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ sudo docker run \
...
+**Note**: If you're running on CentOS, Fedora, or RHEL (or are using LXC), take a look at our [running instructions](docs/running.md).
 
-We have detailed [instructions](docs/running.md#standalone) on running cAdvisor standalone outside of Docker. cAdvisor [running options](docs/runtime_options.md) may also be interesting for advanced usecases. If you want to build your own cAdvisor Docker image see our [deployment](docs/deploy.md) page.
+We have detailed [instructions](docs/running.md#standalone) on running cAdvisor standalone outside of Docker. cAdvisor [running options](docs/runtime_options.md) may also be interesting for advanced usecases. If you want to build your own cAdvisor Docker image, see our [deployment](docs/deploy.md) page.
 
-For [Kubernetes](https://github.com/kubernetes/kubernetes) users, cAdvisor can be run as a daemonset.  See the [instructions](deploy/kubernetes) for how to get started, and for how to kustomize it to fit your needs.
+For [Kubernetes](https://github.com/kubernetes/kubernetes) users, cAdvisor can be run as a daemonset.  See the [instructions](deploy/kubernetes) for how to get started, and for how to customize it to fit your needs.
```
Comment (dashpole) [inline README.md]: Typo is intentional.  [Kustomize](https://github.com/kubernetes-sigs/kustomize#kustomize) is a kubernetes yaml customization tool.

**Comment by dashpole** [comment]
looks good other than my comment

**Pull request merged**